event_sink_clickhouse/tasks.py

```diff
@@ -0,0 +1,30 @@
+"""
+This file contains a management command for exporting course modulestore data to ClickHouse.
+"""
+
+import logging
+
+from celery import shared_task
+from edx_django_utils.monitoring import set_code_owner_attribute
+from opaque_keys.edx.keys import CourseKey
+
+from .sinks.course_published import CoursePublishedSink
+
+log = logging.getLogger(__name__)
+celery_log = logging.getLogger('edx.celery.task')
+
+
+@shared_task
+@set_code_owner_attribute
+def dump_course_to_clickhouse(course_key_string, connection_overrides=None):
```

Comment:
If it's not too expensive to run a full courses dump, can the course_key be optional and dump all courses?

Reply:
The next task for this repo is to add in a management command like the coursegraph one which will handle this use case. It does some additional checking of when the last time a course was dumped and kicks off a celery task for each course, so this shouldn't need to change.